tests/unit/integrations/kserve/test_kserve_step_utils.py

```diff
@@ -0,0 +1,73 @@
+# #  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+# #
+# #  Licensed under the Apache License, Version 2.0 (the "License");
+# #  you may not use this file except in compliance with the License.
+# #  You may obtain a copy of the License at:
+# #
+# #       http://www.apache.org/licenses/LICENSE-2.0
+# #
+# #  Unless required by applicable law or agreed to in writing, software
+# #  distributed under the License is distributed on an "AS IS" BASIS,
+# #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# #  or implied. See the License for the specific language governing
+# #  permissions and limitations under the License.
+
+# import sys
+
+# import pytest
+# from hypothesis import given
+# from hypothesis.strategies import text
+
+# from zenml.exceptions import ValidationError
+
+# if sys.version_info.major == 3 and sys.version_info.minor != 10:
+#     from zenml.integrations.kserve.services.kserve_deployment import (
+#         KServeDeploymentConfig,
```

Comment:
@schustmi @AlexejPenner actually commented these out instead of `@pytest.skip` because including them requires installing kserve on the CI which was the whole problem. If you'd rather I just delete the files completely let me know. I know @safoinme was going to be reworking some kserve things next week in any case, so maybe I can add that back then in collaboration with him?